backend/homework/2/src/main/java/org/calculatesentiment/SentimentAnalyzer.java

```diff
@@ -0,0 +1,97 @@
+package org.calculatesentiment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
```

Comment:
Lacks modularity, try to break down classes for different usecases and then create objects to implement simple logic.